infra/modules/azure_container_app/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "name" {
```

Comment:
I suggest to group everything under the same object and add another object for cae values